src/components/MultiHopTrade/components/TradeInput/components/Claim/hooks/useArbitrumClaimsByStatus.tsx

```diff
@@ -107,10 +97,11 @@ export const useArbitrumClaimsByStatus = (txs: Tx[]) => {
             }
           },
           refetchInterval: 60_000,
+          isEnabled: !props?.isDisabled,
```

Comment:
Nitpick: use `skipToken` instead

Reply:
No skipToken with useQueries when the array isnt statically defined - this is created from a dynamic list of txs so type inference doesnt flow through unless the list of queries is const.